include/private/firebird/common.h

```diff
@@ -19,6 +19,11 @@
 #include <vector>
 #include <algorithm>
 
+#if defined(_MSC_VER)
+#pragma warning(disable:4127)
```

Comment:
This warning ("conditional expression is constant") is pretty useful in general, it would be better to avoid disabling it.